src/plugin.js

```diff
@@ -666,6 +667,10 @@ const contribAdsPlugin = function(options) {
           if (player.paused()) {
             this.state = 'content-set';
           } else {
+            // If we get here we missed the 'play' event, so pause the player so
+            // that content doesn't start playing immediately after the source change
+            player.pause();
```

Comment:
What about if contentupdate for the initial source happens during content-playback? This can happen with autoplay.